string bit_rate
uint8 quality
uint8 max_quality
int8 signal_level